### PR TITLE
refactor, chore, fix(egen): edits google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb notebook

### PR DESCRIPTION
**Pull request by k-root**
- Removes boiler plate
- makes changes as per the template
- adds package versions for dealing with version conflicts 

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb [hunk 1]
```diff
@@ -62,7 +74,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This notebook shows how to use the components defined in [`google_cloud_pipeline_components`](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) in conjunction with an experimental `evaluation` method, to build a [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines) workflow that uploads a tabular custom model as a `Model` resource, creates a `BatchPredictionJob` resource, and evaluates the `Model` resource with the `BatchPredictionJob` results to create an evaluation `system.Metrics` artifact. \n",
+        "This notebook shows how to use the components defined in [`google_cloud_pipeline_components`](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) in conjunction with an experimental `evaluation` method, to build a [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines) workflow that uploads a tabular custom model as a Model resource, creates a BatchPredictionJob resource, and evaluates the Model resource with the BatchPredictionJob results to create an evaluation `system.Metrics` artifact. \n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:77]: change: "Model resource" to "model resource" (twice)

File: notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb [hunk 2]
```diff
@@ -85,9 +97,9 @@
         "\n",
         "The steps performed include:\n",
         "\n",
-        "- Upload a pre-trained model as a `Model` resource.\n",
-        "- Run a `BatchPredictionJob` on the `Model` resource with ground truth data.\n",
-        "- Generate evaluation `Metrics` artifact about the `Model` resource.\n",
+        "- Upload a pre-trained model as a Model resource.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:100]: - change "pre-trained" to "pretrained"
- change "Model resource" to "model resource"

File: notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb [hunk 3]
```diff
@@ -85,9 +97,9 @@
...
+        "- Run a BatchPredictionJob on the Model resource with ground truth data.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:101]: - change "Model resource" to "model resource"

File: notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb [hunk 4]
```diff
@@ -127,253 +139,218 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "create_bucket"
+        "id": "timestamp"
       },
       "source": [
-        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
+        "#### UUID\n",
+        "\n",
+        "If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a uuid for each instance session, and append it onto the name of resources you create in this tutorial."
```
Comment (kittyabs) [inline notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb:333]: - change "you are" to "you're"

**Review by kittyabs — COMMENTED** [review]
numerous edits

**Comment by k-root** [comment]
Addressed the above comments. please review. Thanks!

**Review by katiemn — APPROVED** [review]

**Pull request merged**